src/pages/omOss.tsx

```diff
@@ -0,0 +1,253 @@
+  ],
+  [
+    imagesPath + "mimmiPic.jpg",
+    imagesPath + "dilanPic.jpg",
+  ],
+];
+
+const ecoNames = [["Christofer Gärtner", "Melissa Arslan"]];
+
+const managers = [[imagesPath + "axelPic.jpg", imagesPath + "johanPic.jpg"]];
+const econonmyTeam = [
+  [imagesPath + "christoferPic.jpg", imagesPath + "melissaPic.jpg"],
+];
+
+const managersName = [["Axel Johansson", "Johan Abdi"]];
+
+const prNames = [
+  ["Mathias Magnusson", "Olof", "Maria Tcyshkova"],
+  ["Jakob Petterson", "Adam Egnell", "Anna Yang"],
+  ["Rikaz Nismi", "Mary Abeysekera", "Ernest Umeh"],
+];
+
+const salesNames = [
+  ["William Nordwall", "Linus Markström", "Vilmer"],
+  ["Alex", "Roger", "Elin"],
+  ["Toshide"],
+];
+
+const massNames = [
+  ["Aziz Ali", "Oscar Witt", "Mortada Nasser"],
+  ["Mimmi Weng", "Dilan Ismail"],
+];
+
+function Team({
+  team,
+  reverse,
+  teamPic,
+  names,
+  teamName,
+  teamRoles,
+}: {
+  team: string[][];
+  reverse: boolean;
+  teamPic: string;
+  names: string[][];
+  teamName: string;
+  teamRoles: string[][];
+}) {
+  const numMembers = team.length - 1;
+  const numRows = numMembers / 3;
+  const arr = Array.from({ length: numRows }, (_, index) => index);
+
+  return (
+    <div
+      className={`${
+        reverse ? "flex-row-reverse" : "flex-row"
+      } flex mt-[100px] px-[20px] lg:px-[100px] flex-col lg:flex-row justify-center`}
+    >
+      <div className="flex flex-col">
+        <img src={teamPic} className=""></img>
+        <div className="bg-white/80 lg:bg-white  flex items-center justify-center h-full">
+          <p className="text-4xl text-cerise font-bold lg:px-[100px]">{teamName}</p>
+        </div>
+      </div>
+
+      <div className=" lg:w-[1200px] bg-white/80 flex flex-col overflow-x-auto overflow-y-visible lg:h-[900px] lg:px-1 lg:py-10">
+        {team.map((row, i) => (
+          <div className="flex flex-row justify-center gap-2">
+            {row.map((image, j) => (
+              <div className="flex flex-col items-center justify-center">
+                <img src={image} className="h-[150px] lg:h-auto lg:w-[150px] rounded-[20px]"></img>
+                <p className="text-center text-xs lg:text-sm font-normal"> {names[i][j]}</p>
+                <p className="text-center text-xs lg:text-sm font-normal text-darkblue">{teamRoles[i][j]}</p>
+              </div>
+            ))}
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+}
+
+export default function AboutUs() {
+  const t = useLocale();
+  return (
+    <div className="pt-[200px] pb-[300px]">
+      <h1 className="uppercase text-cerise text-5xl font-medium text-center">
+        {" "}
+        {t.aboutUs.header}
+      </h1>
+      {/*
+        Desktop version
+      */}
+      <div className="flex flex-row items-center justify-center px-[200px] hidden lg:block">
+        <div className="px-[100px] mt-36 bg-[url('/img/grayKth.jpg')] bg-cover bg-no-repeat bg-center h-[600px] w-full flex items-center justify-center gap-5">
```

Comment:
This can be a bit hard to see for people with visual impairment. We should make the text more visible
<img width="1281" alt="image" src="https://github.com/datasektionen/ddagen/assets/66337956/9b7c31cf-fe15-4f56-8685-096001963c00">